test/e2e_tests/data_validation_test.go

```diff
@@ -54,6 +54,7 @@ import (
 var dataValidationMntPathPtr string
 var dataValidationTempPathPtr string
 var dataValidationAdlsPtr string
+var dataValidationFileSharePtr string
```

Comment:
```suggestion
var fileSharePtr string
```

Reply:
Why is it better to have it be fileSharePtr over dataValidationFileSharePtr? Isn't it better to do the latter for consistency with the other ptr variable names?

Reply:
its fine to be consistent